docs/reference/snapshot-restore/apis/get-snapshot-status-api.asciidoc

```diff
@@ -112,18 +121,14 @@ and, when using cloud storage, incur high processing costs.
 ==== {api-path-parms-title}
 
 `<repository>`::
-(Required, string)
-Snapshot repository name used to limit the request. Wildcard (`*`) expressions are supported.
-+
-Use `_current` after the repository name to limit the request only to the currently running snapshots. This usage is preferred to return only snapshots that are currently running and not degrade response time.
+(Optional, string)
+Snapshot repository name used to limit the request. Wildcards (`*`) are not
+supported.
```

Comment:
This is actually a little bit tricky. The current docs indicate wildcards are supported. However, using something like `*` or `my_repo*` doesn't resolve. `_all` (the default) only works with `GET _snapshot/_all/_status`. However, that's equivalent to `GET _snapshot/_status`.

Let me know if we'd like to document `_all` given the caveats. Thanks!

Reply:
Actually I'm wrong. This supports wildcards if the `<snapshot>` isn't specified.